content/en/docs/ambient/overview/APIs/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Ambient APIs
+description: Understanding ambient APIs and its status.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+This page lists all the user facing APIs relevant to the ambient mode and its feature status.
+
+## Labels
+
+| Name | Status |
+| ------------------------------- | -------------------------- |
+| `istio.io/dataplane-mode` | Beta |
```

Comment:
Similar to comment below: if this is GA documentation, these labels should be GA

Reply:
I would drop this and annotations, its already documented on a dedicated page (https://github.com/istio/api/pull/3307 is important though)

Reply:
My intent is to have everything in one page while linking to the official API reference page, so users don't need to go through 100 labels/annotations to identify which ones (in fact only a few of them) are ambient related.

Reply:
Decided to just keep the few important labels. As far as status for the label, I'll add a discussion item for today's meeting. I do feel they are GA also but John had beta in https://github.com/istio/api/pull/3307 so I started with beta.